doc/syntax.bnf

```diff
@@ -122,6 +122,7 @@ reserved "compute"
   | "focus" <nat_lit>
   | "print"
   | "proofterm"
+  | "why3" <ident>?
```

Comment:
The documentation is not consistent with the grammar. In the documentation, you use <string_lit>.

Reply:
Fixed in a29414085332857be96b62ab67a96960b46d746d, it's `<string_lit>` now.